### ISSUE TITLE
miniupnpd-nftables: use 'inet' table for nat rules. But the 'inet' family con't set nat rule both ipv4 and ipv6

### ISSUE THREAD

**Issue by msylgj** [miniupnp/miniupnp#603]
The 'inet' family of nftables con't set nat rule both ipv4 and ipv6. But I con't use ip&ip6 nat table for rule sets.(miniupnpd doesn't support, throw errors: send_batch failed -4) So we can see only ipv4 redirect rules in the nat table. The ruleset table like this: 
```
table inet miniupnpd_filter {
	chain forward {
		type filter hook forward priority -25; policy accept;
		jump miniupnpd
	}

	chain miniupnpd {
		iif "pppoe-wan" th dport 6881 @nh,128,32 0x<ipv4 public ip> @nh,72,8 0x6 accept
		iif "pppoe-wan" th dport 6881 @nh,192,128 0x<ipv6 public ip> @nh,48,8 0x6 accept
		iif "pppoe-wan" th dport 6881 @nh,128,32 0x<ipv4 public ip> @nh,72,8 0x11 accept
		iif "pppoe-wan" th dport 6881 @nh,192,128 0x<ipv6 public ip> @nh,48,8 0x11 accept
		iif "pppoe-wan" th dport 8083 @nh,128,32 0x<ipv4 public ip> @nh,72,8 0x6 accept
		iif "pppoe-wan" th dport 8083 @nh,192,128 0x<ipv6 public ip> @nh,48,8 0x6 accept
	}
}
table inet miniupnpd_nat {
	chain prerouting {
		type nat hook prerouting priority dstnat; policy accept;
		jump prerouting_miniupnpd
	}

	chain postrouting {
		type nat hook postrouting priority srcnat; policy accept;
		jump postrouting_miniupnpd
	}

	chain prerouting_miniupnpd {
		iif "pppoe-wan" @nh,72,8 0x6 th dport 6881 dnat ip to 192.168.2.3:6881
		iif "pppoe-wan" @nh,72,8 0x11 th dport 6881 dnat ip to 192.168.2.3:6881
		iif "pppoe-wan" @nh,72,8 0x6 th dport 8083 dnat ip to 192.168.2.3:8083
	}

	chain postrouting_miniupnpd {
	}
}
```

**Comment by sjnewbury** [comment]
Was this working for you before https://github.com/miniupnp/miniupnp/pull/562 was merged?  My firewall was broken by https://github.com/miniupnp/miniupnp/pull/562, I've had to revert to the version before the merge for now until I can figure out what is going on.   My firewall setup maintains separate ip and ip6 nft rules and has been working great up until now.

**Comment by msylgj** [comment]
> Was this working for you before #562 was merged? My firewall was broken by #562, I've had to revert to the version before the merge for now until I can figure out what is going on. My firewall setup maintains separate ip and ip6 nft rules and has been working great up until now.

I haven't any broken of firewall yet, but no ipv6 rules. I will have a try to revert #562, thanks for you information.
